src/index.ts

```diff
@@ -143,14 +144,18 @@ declare interface Props {
   linkify?: boolean;
   className?: string;
   useClasses?: boolean;
+  as?: string;
+  anser?: Anser;
 }
 
 export default function Ansi(props: Props): JSX.Element {
-  const { className, useClasses, children, linkify } = props;
+  const { className, useClasses, children, linkify, as, anser } = props;
+  const elementType = ( as === "span" ) ? React.Fragment : "code";
+  const elementProps = ( as === "span" ) ? null : { className };
   return React.createElement(
-    "code",
-    { className },
-    ansiToJSON(children ?? "", useClasses ?? false).map(
+    elementType,
+    elementProps,
+    ansiToJSON(children ?? "", useClasses ?? false, anser ?? new Anser()).map(
```

Comment:
This is a big no-no in React because it means that your renders are no longer pure (the component ends up mutating the internal state of the `Anser` instance).